systemtest/src/test/java/io/strimzi/systemtest/kafka/dynamicconfiguration/DynamicConfigurationSharedST.java

```diff
@@ -179,6 +186,41 @@
         return testCases;
     }
 
+    /**
+     * Method, which randomly choose 3 dynamic properties for verification from @see{testCases}. In this case we are ok
+     * with stochastic selection, because we don't care, which configuration is used. Furthermore, it's the same path
+     * of code (i.e., same CFG (control flow graph), which does not triggers RollingUpdate).
+     * @param testCases test cases, where each consist of one dynamic property
+     * @return List 3 chosen dynamic properties
+     */
+    private static List<String> stochasticSelection(final Map<String, Object> testCases) {
+        final List<String> testCaseKeys = new ArrayList<>(testCases.keySet());
+        final Random generator = new Random();
```

Comment:
Shouldn't this be a class member?

Reply:
In this case, it does not matter because this pseudo-generator is instantiated only once and does not breaks the mathematical foundation. (i.e., do not depends on seed, which is not random). But I will move it to class if in the future one will use it in another part of code...